vnext/Microsoft.ReactNative/ReactApplication.h

```diff
@@ -95,11 +97,61 @@ struct ReactApplication : NoDefaultCtorReactApplication_base<ReactApplication> {
   void OnSuspending(IInspectable const &, Windows::ApplicationModel::SuspendingEventArgs const &);
   void OnNavigationFailed(IInspectable const &, xaml::Navigation::NavigationFailedEventArgs const &);
 
+  using AppLaunchedDelegate = winrt::delegate<void(
+      winrt::Microsoft::ReactNative::ReactApplication sender,
+      winrt::Windows::ApplicationModel::Activation::LaunchActivatedEventArgs args)>;
+
+  void LaunchedInternal(AppLaunchedDelegate del) {
+    m_launched = del;
+  }
+  AppLaunchedDelegate LaunchedInternal() {
+    return m_launched;
+  }
+
+  using AppViewCreatedDelegate =
+      winrt::delegate<void(winrt::Microsoft::ReactNative::ReactApplication sender, winrt::hstring args)>;
+
+  void ViewCreatedInternal(AppViewCreatedDelegate del) {
+    m_viewCreated = del;
+  }
+
+  AppViewCreatedDelegate ViewCreatedInternal() {
+    return m_viewCreated;
+  }
+
+  using AppPageNavigatedDelegate =
+      winrt::delegate<void(winrt::Microsoft::ReactNative::ReactApplication sender, ReactRootView view)>;
+
+  void PageNavigatedInternal(AppPageNavigatedDelegate del) {
+    m_pageNavigated = del;
+  }
+
+  AppPageNavigatedDelegate PageNavigatedInternal() {
+    return m_pageNavigated;
+  }
+
+  XamlMetaDataProvider m_provider;
```

Comment:
>m_provider [](http://example.com/codeflow?start=23&length=10)

Please re-arrange code in this file:
- fields must go below in the private section wit other fields.
- Method implementation must go to .cpp file. Having implementation makes compilation slower. And this case differs from the previous style this file had.